time stamp
int32[4] rpm